mocks/mocks.go

```diff
@@ -394,13 +406,15 @@ func (sa *StorageAuthorityReadOnly) GetOrder(_ context.Context, req *sapb.OrderR
 		return nil, errors.New("very bad")
 	}
 
-	created := sa.clk.Now().AddDate(-30, 0, 0).Unix()
-	exp := sa.clk.Now().AddDate(30, 0, 0).Unix()
+	created := sa.clk.Now().AddDate(-30, 0, 0)
+	exp := sa.clk.Now().AddDate(30, 0, 0)
 	validOrder := &corepb.Order{
 		Id:                req.Id,
 		RegistrationID:    1,
-		CreatedNS:         created,
-		ExpiresNS:         exp,
+		CreatedNS:         created.Unix(),
```

Comment:
Pretty sure these should say `.UnixNano()`, and it was a bug for the previous version of this test to use `.Unix()`.

Reply:
I get some test errors changing that, perhaps that type of cleanup should be another ticket?
```
--- FAIL: TestFinalizeOrder (0.02s)
    --- FAIL: TestFinalizeOrder/Good_CSR,_Ready_Order (0.00s)
        wfe_test.go:2736: Field "expires": Expected "1970-01-01T00:00:00.9466849Z", got "2000-01-01T00:00:00Z"
--- FAIL: TestGetOrder (0.01s)
    --- FAIL: TestGetOrder/Good_request (0.00s)
        wfe_test.go:2968: Field "expires": Expected "1970-01-01T00:00:00.9466848Z", got "2000-01-01T00:00:00Z"
```

Reply:
Yeah those tests are failing because their [hardcoded expectations](https://github.com/letsencrypt/boulder/blob/7a91938865b747654d2322faaa694058ae33abed/wfe2/wfe_test.go#L2702) have [clearly nonsensical](https://github.com/letsencrypt/boulder/blob/7a91938865b747654d2322faaa694058ae33abed/wfe2/wfe_test.go#L2875) timestamps. Less than a second after the unix epoch? It seems clear to me that those test expectations were created by copying the resulting timestamp from a debugging log line without realizing that the mock was just doing the wrong thing.

Totally fine to address this in a separate PR.

Reply:
Can confirm that is probably how the test expectations were created.